engine/source/runtime/function/render/render_pipeline_base.h

```diff
@@ -14,6 +14,7 @@ namespace Pilot
 
     struct RenderPipelineInitInfo
     {
+        bool enable_fxaa;
```

Comment:
need a default variable initialization bool enable_fxaa {false};

Reply:
well, i will set the default variable, thanks